src/main/java/rest/demo/service/TaskService.java

```diff
@@ -0,0 +1,33 @@
+package rest.demo.service;

```

Comment:
Don't need this!

Reply:
Can you please show me how would you implement tasks without task service?